### DISCUSSION TITLE
Redynamisation du top pour les nouveaux

### DISCUSSION THREAD

**Discussion by leuhvraiminoryo** [Crownicles/Crownicles#2345] [Nouvelles idées]
Un des problèmes majeurs de draftbot à mes yeux (à égalité avec la triste absence de commu anglophone), est à quel point, de par l'effet du temps et de l'accumulation qui a pu être réalisée par certains, il est long et douloureux pour de nouveaux joueurs de monter dans le classement principal du jeu. Je pense même qu'il s'agit d'une des raisons qui amènent une si grande quantité de joueurs à stopper peu après avoir commencé (d'ailleurs à ce propos, il faudrait tenter de mettre plus en avant les missions auprès des nouveaux). 

Voilà donc une proposition pour tenter de pallier à ça sans pour autant réellement affecter négativement qui que ce soit, ni être trop coûteuse à développer (normalement) (même si ça devrait possiblement attendre après draftbot v5 uh).

Je propose que, après qu'une personne soit "inactive" depuis un assez fort temps (à fixer, mais sans aucun doute de l'ordre des mois), elle se voit affecter une altération d'état particulière. Une personne ayant cette altération serait alors tout simplement ignorée dans le calcul du top. Cela permet de prune out pas mal d'anciens actifs et semi-actifs, tout en bien sûr ne changeant rien pour eux lors de leur possible retour.

Je pense aussi qu'à cet effet, utiliser l'alté :ghost: , et simplement repousser la date à laquelle elle est mise, serait probablement mieux que créer une autre alté à effet similaire mais juste durée d'enclanchement plus grande.

Bref, bisous bisous et j'aimerais vraiment que vous considériez ceci, ou que vous proposiez d'autres façons de régler ce problème.

**Comment by voltou91** [comment]
Me viens une idée concernant cette alté, pour pousser les gens à revenir on pourrait faire un système qui quand tu reviens après x temps tu gagnes x récompenses/points. Je prend ici l'exemple de clash of clans où quand ton compte est inactif pendant longtemps et que tu reviens, tes villageois ont amélioré ton village (plus t'es parti longtemps, plus il y a d'améliorations faites) mais ça reste plus rentable de jouer que de rester inac pour améliorer un peu ton village. Donc ici ce serait plutôt le roi qui t'envoie un cadeau pour ton retour dans ta quête, on pourrait gagner de l'expériences, des points, un item, des gemmes, de l'or, etc...

**Reply by BastLast** [reply]
Go faire une suggestion temporaire pr ton truc voltou

**Comment by BastLast** [comment]
> Un des problèmes majeurs de draftbot à mes yeux (à égalité avec la triste absence de commu anglophone), est à quel point, de par l'effet du temps et de l'accumulation qui a pu être réalisée par certains, il est long et douloureux pour de nouveaux joueurs de monter dans le classement principal du jeu. Je pense même qu'il s'agit d'une des raisons qui amènent une si grande quantité de joueurs à stopper peu après avoir commencé (d'ailleurs à ce propos, il faudrait tenter de mettre plus en avant les missions auprès des nouveaux). 
> 
> Voilà donc une proposition pour tenter de pallier à ça sans pour autant réellement affecter négativement qui que ce soit, ni être trop coûteuse à développer (normalement) (même si ça devrait possiblement attendre après draftbot v5 uh).
> 
> Je propose que, après qu'une personne soit "inactive" depuis un assez fort temps (à fixer, mais sans aucun doute de l'ordre des mois), elle se voit affecter une altération d'état particulière. Une personne ayant cette altération serait alors tout simplement ignorée dans le calcul du top. Cela permet de prune out pas mal d'anciens actifs et semi-actifs, tout en bien sûr ne changeant rien pour eux lors de leur possible retour.
> 
> Je pense aussi qu'à cet effet, utiliser l'alté :ghost: , et simplement repousser la date à laquelle elle est mise, serait probablement mieux que créer une autre alté à effet similaire mais juste durée d'enclanchement plus grande.
> 
> Bref, bisous bisous et j'aimerais vraiment que vous considériez ceci, ou que vous proposiez d'autres façons de régler ce problème.

Je trouve ca cool que les gens soient dans le top a jamais comme des formes de milestones

Et les points sont de plus en plus simple a avoir donc les anciens qui jouent pas vont progressivement disparaître

**Reply by leuhvraiminoryo** [reply]
Si tu veux qu'ils puissent être en "milestones", autant rajouter un paramètre à la commande top pour les voir quand même si vraiment on veut - il n'empêche que même avec le gain de points plus facile, les joueurs prennent plusieurs mois même en étant actifs à entrer dans le top 1k, alors même qu'il n'y a pas du tout 1k de joueurs actifs sur la période concernée. De ce que j'ai pu voir être écrit, ça en embête plus d'un.
